snmp/datadog_checks/snmp/snmp.py

```diff
@@ -489,6 +489,8 @@ def report_metrics(
                 for index, val in iteritems(results[name]):
                     metric_tags = tags + self.get_index_tags(index, results, metric.index_tags, metric.column_tags)
                     self.submit_metric(name, val, metric.forced_type, metric_tags, metric.options)
+                    if self.is_bandwidth_metric(name):
+                        self.submit_bandwidth_usage_metric(name, index, results, metric_tags)
```

Comment:
I think it's safer if we wrap `self.submit_bandwidth_usage_metric(name, index, results, metric_tags)` in a `try/except Exception` clause to catch any exception. We want to prevent the integration failing if an exception is raised at `self.submit_bandwidth_usage_metric`. 

Note that, similar `try/except Exception` protection is used in `self.submit_metric` method.